content/en/docs/setup/getting-started/index.md

```diff
@@ -44,7 +44,10 @@ Follow these steps to get started with Istio:
     The command above downloads the latest release (numerically) of Istio.
     To download a specific version, you can add a variable on the command line.
     For example to download Istio 1.4.3, you would run
-      `curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.4.3 sh -`
+    `curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.6.8 sh -`. The
```

Comment:
You need to update the 1.4.3 in the prior sentence.

Reply:
Thanks. Fixed.